image_services/glo_30_hand/make_hand_service.py

```diff
@@ -0,0 +1,173 @@
+    in_mosaic_dataset=mosaic_dataset,
+    raster_type='Raster Dataset',
+    input_path='/vsis3/glo-30-hand/v1/2021/',
+    filter='REGEX:.*Copernicus_DSM_COG_10_[NS][0-8]\d_00_[EW]\d\d\d_00_HAND.tif',
+)
+
+arcpy.management.CalculateFields(
+    in_table=mosaic_dataset,
+    fields=[
+        ['Tile', '!Name!.split("_")[4] + !Name!.split("_")[6]'],
+        ['Tag', '"GLO30_HAND"'],
+        ['Dataset_ID', '"Global_30m_HAND"'],
+        ['ProductName', '"GLO30_HAND_"+ !Name!.split("_")[4] + !Name!.split("_")[6]'],
+        ['URLDisplay', '"GLO30_HAND_"+ !Name!.split("_")[4] + !Name!.split("_")[6]'],
+        ['DownloadURL', '"https://glo-30-hand.s3.amazonaws.com/v1/2021/" + !Name! + ".tif"'],
+        ['MaxPS', '610'],
+    ],
+)
+
+arcpy.management.BuildFootprints(
+    in_mosaic_dataset=mosaic_dataset,
+    reset_footprint='NONE',
+    min_data_value=0,
+    max_data_value=1,
+    approx_num_vertices=12,
+    update_boundary='UPDATE_BOUNDARY',
+)
+
+arcpy.management.BuildBoundary(
+    in_mosaic_dataset=mosaic_dataset,
+    append_to_existing='OVERWRITE',
+    simplification_method='NONE',
+)
+
+arcpy.management.SetMosaicDatasetProperties(
+    in_mosaic_dataset=mosaic_dataset,
+    rows_maximum_imagesize=5000,
+    columns_maximum_imagesize=5000,
+    allowed_compressions='JPEG;NONE;LZ77',
+    default_compression_type='JPEG',
+    JPEG_quality=80,
+    resampling_type='NEAREST',
+    LERC_Tolerance=0.01,
+    clip_to_footprints='CLIP',
+    clip_to_boundary='CLIP',
+    color_correction='NOT_APPLY',
+    footprints_may_contain_nodata='FOOTPRINTS_MAY_CONTAIN_NODATA',
+    allowed_mensuration_capabilities='BASIC',
+    default_mensuration_capabilities='BASIC',
+    allowed_mosaic_methods='Center;NorthWest;Nadir;LockRaster;ByAttribute;Seamline;None',
+    default_mosaic_method='ByAttribute',
+    order_field='Name',
+    sorting_order='Ascending',
+    mosaic_operator='FIRST',
+    blend_width=10,
+    view_point_x=300,
+    view_point_y=300,
+    max_num_per_mosaic=50,
+    cell_size_tolerance=1.8,
+    cell_size=3,
+    metadata_level='BASIC',
+    transmission_fields='Name;MinPS;MaxPS;LowPS;HighPS;ZOrder;Dataset_ID;CenterX;CenterY;Tag;Tile;ProductName;DownloadURL;URLDisplay',
+    use_time='DISABLED',
+    max_num_of_download_items=50,
+    max_num_of_records_returned=2000,
+    processing_templates=f'{raster_function_template};None',
+    default_processing_template=raster_function_template,
+)
+
+arcpy.management.CalculateCellSizeRanges(
+    in_mosaic_dataset=mosaic_dataset,
+    do_compute_min='NO_MIN_CELL_SIZES',
+    do_compute_max='NO_MAX_CELL_SIZES',
+    max_range_factor=10,
+    cell_size_tolerance_factor=0.8,
+    update_missing_only='UPDATE_ALL',
+)
+
+with arcpy.EnvManager(cellSize=600):
+    arcpy.management.CopyRaster(
+        in_raster=mosaic_dataset,
+        out_rasterdataset=local_overview,
+    )
+
+subprocess.run(['aws', 's3', 'cp', local_overview, s3_overview.replace("/vsis3/", "s3://"), '--recursive'])
```

Comment:
We could use `boto3` but I support the decision to use the `aws` cli if it makes this significantly simpler.

Reply:
Not blocking.